### PR TITLE
Async Scheduler

### PR DESCRIPTION
**Pull request by corps**
No ready for review, but checking in against losing local work.

### CODE REVIEW

File: etna/docker-compose.yml [hunk 1]
```diff
@@ -22,6 +22,7 @@ services:
       - ../docker/etna-base/entrypoints:/entrypoints:ro
       - /etc/passwd:/etc/passwd:ro
       - /etc/group:/etc/group:ro
+      - $HOME/etna.yml:/root/etna.yml
```
Comment (corps) [inline etna/docker-compose.yml:25]: Not, entirely convinced this is wise.  If you don't have etna.yml when you start this up, it creates a directory lol.

File: etna/lib/etna/command.rb [hunk 2]
```diff
@@ -203,6 +203,7 @@ def subcommands
       @subcommands ||= self.class.constants.sort.reduce({}) do |acc, n|
         acc.tap do
           c = self.class.const_get(n)
+          next unless c.respond_to?(:instance_methods)
```
Comment (corps) [inline etna/lib/etna/command.rb:206]: Unrelated bug I found in the etna gem, due to using a const that wasn't another command.

File: etna/docker-compose.yml (hunk 1 above)
Comment (graft) [reply etna/docker-compose.yml:25]: Why would you want to mount etna.yml anyway?

Comment (corps) [reply etna/docker-compose.yml:25]: Eh, for testing the etna command stuff more conveniently.

Comment (graft) [reply etna/docker-compose.yml]: oh ah - yeah i have been just running gem build/install for testing, but it is painful.

**Pull request merged**